docs/reference/two_step_validation.rst

```diff
@@ -35,12 +35,12 @@ Edit the configuration file:
         google_authenticator:
             enabled: true
             server:  yourserver.com
-            ip_white_list:
+            trusted_ip_list:
```

Comment:
You could make this BC by adding a new configuration key, and deprecate the old one in favor of the new one, which will make upgrading easier.